### PR TITLE
syncback: docs

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @nicks,

Please review the following commits I made in branch maiamcc/syncback-docs:

06621853de7cd8b8bd157cb581b8ba45c9da73ef (2020-12-09 17:06:11 -0500)
add image

dea9a480b9141e86d0dcf64e3a24e246abe918a6 (2020-12-09 16:56:51 -0500)
syncback: docs

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: syncback/README.md [hunk 1]
```diff
@@ -2,14 +2,95 @@
...
+## Overview
+This extension provides a function `syncback` which creates a manually-triggered local resource to sync files from a Kubernetes container back to your local filesystem. You can configure the source pod/container/directory/files+directories, and the destination location, as well as some other settings.
 
-## Common Use Cases
-tk
+Whenever you want to sync files back from the pod to your local filesystem, invoke the local resource from the Web UI:
 
-## Advanced Usage
-tk
+![A syncback resource in the sidebar, with the trigger button highlighted](assets/trigger-syncback.png)
```
Comment (nicks) [inline syncback/README.md:10]: this is a good place to call out `tilt trigger` too! :grin:

File: syncback/README.md [hunk 2]
```diff
@@ -2,14 +2,95 @@
...
+
+### Usage
+First, **make sure you have an up-to-date `rsync` on your local machine** (version >= 3.0.0).
```
Comment (nicks) [inline syncback/README.md:13]: i wonder if the extension should check if `rsync` exists on load...

**Review by nicks — APPROVED** [review]
fwiw, my "technical writer" alarm bells always go off whenever i see a README that has a big wall of text of caveats and technical detail. 

I suspect the intro user doesn't really care that much how it's implemented, and cares a lot more about how to make it work for their project.

wonder if some of this should be moved to an ARCHITECTURE.md or smth

File: syncback/README.md (hunk 2 above)
Comment (maiamcc) [reply syncback/README.md:13]: way ahead of you! https://github.com/tilt-dev/tilt-extensions/pull/113

**Comment by maiamcc** [comment]
> fwiw, my "technical writer" alarm bells always go off whenever i see a README that has a big wall of text of caveats and technical detail.
> 
> I suspect the intro user doesn't really care that much how it's implemented, and cares a lot more about how to make it work for their project.
> 
> wonder if some of this should be moved to an ARCHITECTURE.md or smth

that... is super reasonable. I'll move this around.

**Pull request merged**